final/60_telegraf_cleanup.sh

```diff
@@ -0,0 +1,2 @@
+# Restart telegraf to ensure a clean start after any hostname changes
+systemctl restart telegraf
```

Comment:
```suggestion
systemctl restart telegraf

```

Reply:
Like this?

Reply:
Yeah, I assume it's needed as git was making a point of it when I looked earlier.